pkg/volume/flexvolume/attacher-defaults.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flexvolume
+
+import (
+	"fmt"
+	"path"
+	"time"
+
+	"github.com/golang/glog"
+
+	"k8s.io/kubernetes/pkg/util/mount"
+	"k8s.io/kubernetes/pkg/volume"
+)
+
+type attacherDefaults flexVolumeAttacher
+
+// Attach is part of the volume.Attacher interface
+func (a *attacherDefaults) Attach(spec *volume.Spec, hostName string) (string, error) {
+	glog.Warningf("Using default Attach")
```

Comment:
Extra credit: add identifying information in the log messages (e.g. volume name, pod name/uid when available, etc.) so it's easier to associate a log message with a particular pod or volume.